requirements.txt

```diff
@@ -2,6 +2,6 @@ lightkube
 ops
 ops.manifest>=1.1.3,<2.0.0
 pydantic==1.*
-ops.interface-kube-control @ git+https://github.com/charmed-kubernetes/interface-kube-control.git@ops-0.2.0#subdirectory=ops
+ops.interface-kube_control == 0.2.0
 ops.interface-tls-certificates @ git+https://github.com/charmed-kubernetes/interface-tls-certificates.git@236e089493c96feac5ea1a8617b1059a4477a04e#subdirectory=ops
-ops.interface_openstack_integration @ git+https://github.com/charmed-kubernetes/interface-openstack-integration.git@d05f2f1a392cabdd15a1ab8348a0e546c32519f7#subdirectory=ops
+ops.interface_openstack_integration @ git+https://github.com/canonical/interface-openstack-integration.git@KU-3229/support-o7k-endpoint-through-proxy#subdirectory=ops
```

Comment:
Needs this merged first::
* https://github.com/canonical/interface-openstack-integration/pull/19